src/setup-fontawesome.js

```diff
@@ -66,6 +66,7 @@ import {
     faSpinner,
     faStar,
     faT,
+    faTags,
```

Comment:
I don't think we are using it after the switch to text/checkbox instead of icon/button, might be possible to remove it